tests/e2e/config/bootstrap.js

```diff
@@ -202,7 +202,7 @@ function observeConsoleLogging() {
 		//
 		// We ignore them as they are not indicative of a problem with the
 		// test and usually make E2E tests fail erroneously.
-		if ( text.startsWith( 'You are probably offline.' ) ) {
+		if ( text.includes( 'You are probably offline.' ) ) {
```

Comment:
Noting here this is all good as we discussed 👍 The reason being that we need to account for the same error which may be secondarily logged through our API module that will wrap it in a way that the `text` would not start with the message anymore, but we'd still want to ignore it.